admin_guide/pruning_resources.adoc

```diff
@@ -615,3 +615,29 @@ finished, the registry can be switched back to read-write mode by executing:
 $ oc env -n default dc/docker-registry REGISTRY_STORAGE_MAINTENANCE_READONLY-
 ----
 endif::[]
+
+[[pruning-cronjobs]]
+== Pruning Cron Jobs
+
+[IMPORTANT]
+====
+Cron Jobs is a Technology Preview feature only.
+ifdef::openshift-enterprise[]
+Technology Preview features are not
+supported with Red Hat production service level agreements (SLAs), might not be
+functionally complete, and Red Hat does not recommend to use them for
+production. These features provide early access to upcoming product features,
+enabling customers to test functionality and provide feedback during the
+development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+At this time, the results of cron jobs are not automatically pruned.
+Therefore, cluster administrator should perform regular
+xref:../../dev_guide/cron_jobs.adoc#cleaning-up-after-a-cron-job[cleanup of jobs], manually.  We also recommend to
+xref:manage_authorization_policy.adoc[restrict the access] to corn jobs to a
```

Comment:
funny typo :) it's cron not corn

Reply:
ha, and I nailed it somewhere else and got it wrong here ;) but I can blame @gaurav-nelson he made a type from where I copy pasted ;)

Reply:
Fixed.